### PR TITLE
Feature/theme

### PR DESCRIPTION
**Pull request by BradleyBuurmeester**

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -28,3 +28,12 @@ body{
...
   width: 40% !important;
 }
+
+//Theme variables
+:root{
+  --primary-bg-color: #750C00;
+  --secondary-bg-color: #a01101;
+  --secondary-bg-hover-color: #881103;
+  --primary-txt-color: white;
+  --primary-hover-txt-color: lightgrey;
```
Comment (NoNamer777) [inline src/styles.scss]: Maybe it's a good idea to think about consistency. Either we use hexadecimals in the whole project or the name of a color

Comment (BradleyBuurmeester) [reply src/styles.scss]: true

File: src/app/components/content/home/home.component.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <div class="text-center">
     <p class="font-weight-bolder h3 mb-5 mx-4" id="intro-text">Last gehad van (seksuele) intimidatie tijdens het uitgaan?</p>
     <button
-      class="font-weight-bolder btn btn-lg btn-light rounded-pill border border-dark shadow mt-5"
```
Comment (NoNamer777) [inline src/app/components/content/home/home.component.html:5]: any particular reason why you removed the rounded corners and font-weight styling?

Comment (BradleyBuurmeester) [reply src/app/components/content/home/home.component.html:5]: I styled the button the same way I styled it in the partner component for consistency. The rounded corners are added with the "btn" class.

Comment (NoNamer777) [reply src/app/components/content/home/home.component.html:5]: > The rounded corners are added with the "btn" class.

I know that, 

I'm just wondering why you adjusted the styling of a button that was already conform styling that Rosanne already had provided. Also, checking the designs for the partner page also shows some deviations between the designs provided by Rosanne and our implementation.

**Review by NoNamer777 — APPROVED** [review]
Overall, looks good. But I think we really need to consider how much we're going to deviate from the designs that Rosanne provides. That she's doubting about what color scheme, we can wait on that and fill in temporarily until she decides, but other things like form and shape, we can follow, imo.

**Pull request merged**